package-system/AWSGameLiftServerSDK/README.md

```diff
@@ -3,115 +3,30 @@
 You can find the official GameLift documentation [here](https://aws.amazon.com/documentation/gamelift/).
 
 ## Minimum requirements:
-* Either of the following:  
-  * Microsoft Visual Studio 2017 - Windows Build
-  * Clang 6.0 - Linux Build
-* CMake version 3.1 or later
-* Python version 3.6 or later
+* Either of the following:
+  * Microsoft Visual Studio 2019 - Windows Build
+  * Clang 12.0 and GCC 9.3 - Linux Build
+* CMake version 3.20 or later
```

Comment:
CMake 3.20.5 or later to be precise.

Assume some of this is driven off Linux min specs for O3DE. Should we link to those docs?